upup/pkg/fi/cloud.go

```diff
@@ -27,6 +27,23 @@ type Cloud interface {
 	ProviderID() CloudProviderID
 
 	DNS() (dnsprovider.Interface, error)
+
+	// FindVPCInfo looks up the specified VPC by id, returning info if found, otherwise (nil, nil)
+	FindVPCInfo(id string) (*VPCInfo, error)
+}
+
+type VPCInfo struct {
```

Comment:
I'd rather do it when we need it, if that's OK?  The intention is not to put AWS structs into this package.